src/schema/objects/suffixes.yaml

```diff
@@ -482,6 +505,10 @@ coordsystem:
   description: |
     A JSON document specifying the coordinate system(s) used for the MEG, EEG,
     head localization coils, and anatomical landmarks.
+dataset_description:
```

Comment:
```suggestion
# This is a special top-level file, which is why an underscore is allowed in the "suffix".
dataset_description:
```

I'm thinking we could just add comments to this one and `genetic_info` so folks don't think they can include underscores in new suffixes.

Reply:
Yeah, I really liked this idea, but as per the prevalent opinion in the discussion today apparently we should just strike it down as a suffix :(

Maybe we can bring the suffixification discussion back later after we get a foot in the door with this ^^

Reply:
Agreed!